internal/rbd/replicationcontrollerserver.go

```diff
@@ -557,7 +557,13 @@ func (rs *ReplicationServer) PromoteVolume(ctx context.Context,
 
 	// promote secondary to primary
 	if !mirroringInfo.Primary {
-		err = rbdVol.promoteImage(req.Force)
+		if req.Force {
```

Comment:
instead of accessing members directly, is a little better to use `req.GetForce()` so show these requests are read-only.